### PR TITLE
To do list Project

### PR DESCRIPTION
**Pull request by Have-Samuel**
**To do list Project**
In this Project We:

1. Use webpack to bundle JavaScript.
2. Learn how to use proper ES6 syntax.
3. Use ES6 modules to write modular JavaScript.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (Dipeshtwis) [inline .gitignore:1]: - [ ] You have put `node_modules` here but still, you have pushed the node_modules. Please remove the node_modules folder from Github.

File: README.md [hunk 2]
```diff
@@ -18,39 +19,19 @@
...
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+1. git clone git@github.com:Have-Samuel/todoList.git
+2. Cd into to-Do-List repo
+3. Use the live server to view the page
```
Comment (Dipeshtwis) [inline README.md]: - [ ] When I run the `live server`, the page looks like this.
Screenshot from 2022-03-24 00-19-34

Please mention the correct step to run the project.

```
# After entering in to the correct folder 

$ npm i
$ npm start
```

- [ ]  Also fix the `Live Demo` section. Please remove that section for now if you don't have a live link to the project.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Have-Samuel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N.A.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**